src/backend/src/routes/teams.routes.ts

```diff
@@ -7,7 +7,13 @@ const teamsRouter = express.Router();
 
 teamsRouter.get('/', TeamsController.getAllTeams);
 teamsRouter.get('/:teamId', TeamsController.getSingleTeam);
-teamsRouter.post('/:teamId/set-members', TeamsController.setTeamMembers);
+teamsRouter.post(
+  '/:teamId/set-members',
+  body('userIds').isArray(),
+  body('userIds.*').isInt({ min: 1 }),
```

Comment:
I might be wrong but I believe we can use the intminzero function here... Anthony might correct me on that.